app/assets/javascripts/modules/CaseClosure.js

```diff
@@ -52,6 +61,17 @@ moj.Modules.CaseClosure = {
     });
   },
 
+  showHideOtherOverturned: function() {
+      var other_is_checked = this.$outcomeReasonOtherOption.is(':checked');
+      var other_option_details_has_value = $('#sar_internal_review_other_option_details').val() ? true : false;
+
+      if (other_is_checked || other_option_details_has_value) {
```

Comment:
So I was asking if we should remove " || other_option_details_has_value" or do you want to show the input just because it has a value in it, even if the checkbox is unchecked.